### PR TITLE
Add system requirements parent docs page for podman and colima

### PR DESCRIPTION
**Pull request by gauravgahlot**
Closes - #554 

Screenshot 2022-10-08 at 10 21 50 PM

### CODE REVIEW

**Comment by gauravgahlot** [comment]
@mdelapenya I'm a bit unsure if this is the only required change. Please take a look and let me  know.

File: mkdocs.yml [hunk 1]
```diff
@@ -52,6 +52,9 @@ nav:
           - examples/cockroachdb.md
           - examples/nginx.md
           - examples/redis.md
+    - System Requirements:
```
Comment (mdelapenya) [inline mkdocs.yml:55]: Could you remove the entries in the features section, to avoid duplication?

Comment (gauravgahlot) [reply mkdocs.yml:53]: Done

**Review by mdelapenya — APPROVED** [review]
LGTM, thanks for your contributions!

**Pull request merged**